### PR TITLE
LRQA 55606 | master

### PR DESCRIPTION
**Pull request by calvinctvo**
https://issues.liferay.com/browse/LRQA-55606

### CODE REVIEW

**Comment by calvinctvo** [comment]
ci:test:relevant

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetrics.testcase [hunk 1]
```diff
@@ -487,6 +487,101 @@ definition {
...
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
+
+		for (var blogNumber : list "1,2,3,4,5,6") {
+			JSONBlog.addEntry(
+				entryContent = "Blogs Entry ${blogNumber} Content",
+				entryTitle = "Blogs Entry Title ${blogNumber}");
+		}
```
Comment (rodrigocunhaa) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetrics.testcase:511]: Create a macro for this part of the code. Similar to `WorkflowMetrics.createAlternativeInstances`.

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/macros/WorkflowMetrics.macro [hunk 2]
```diff
@@ -143,6 +143,21 @@ definition {
 		}
 	}
 
+	macro assertCheckbox {
```
Comment (rodrigocunhaa) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/macros/WorkflowMetrics.macro:146]: Just like you did for the other macro, change it to `assertCheckboxChecked`.

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetrics.testcase [hunk 3]
```diff
@@ -1191,6 +1286,93 @@ definition {
...
+			key_workflowAssetTitle = "Blogs Entry Title 3",
+			locator1 = "MyWorkflowTasks#TABLE_GENERIC_ASSET_TITLE");
+		AssertElementPresent(
+			key_workflowAssetTitle = "Blogs Entry Title 4",
+			locator1 = "MyWorkflowTasks#TABLE_GENERIC_ASSET_TITLE");
+		AssertElementPresent(
+			key_workflowAssetTitle = "Blogs Entry Title 5",
+			locator1 = "MyWorkflowTasks#TABLE_GENERIC_ASSET_TITLE");
+	}
+
```
Comment (rodrigocunhaa) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetrics.testcase:1375]: Is this test really necessary? The first test you created is already covering all the 4 scenarios from [LRQA-55606](https://issues.liferay.com/browse/LRQA-55606). Is this test from another automation task? If not, you can get rid of this.

**Comment by calvinctvo** [comment]
ci:test:relevant

**Comment by rodrigocunhaa** [comment]
Just started reviewing :)

**Comment by rodrigocunhaa** [comment]
Hi @calvinctvo , thank you for that. Regarding the macro, I mentioned creating a new one similar to the `createAlternativeBlogInstances`. Anyway, I've done it here: https://github.com/rodrigocunhaa/liferay-portal/pull/157/commits/20648e4ba213124431c74c320956530a853919e1

Good job and keep up the good work!

**Pull request closed without merging**